pre_award/application_store/db/models/feedback/end_of_application_survey.py

```diff
@@ -19,6 +19,20 @@ class EndOfApplicationSurveyFeedback(BaseModel):
 
     __table_args__ = (db.UniqueConstraint("application_id", "page_number", name="_unique_application_page"),)
 
+    @property
+    def get_section_comment_rating(self) -> tuple[str, str, str]:
+        section = ""
+        comment = ""
+        rating = ""
+        for key in self.data.keys():
+            if key == "more_detail":
+                comment = self.data[key]
+            elif key != "csrf_token":
```

Comment:
Are we storing the csrf_token from the form in the DB? 👀

Reply:
Yes. Yes we are!

Reply:
"cool" 👀